packages/react/src/wallet/wallets/metamaskWallet.ts

```diff
@@ -0,0 +1,18 @@
+import type { WalletOptions, Wallet } from "@thirdweb-dev/react-core";
+import { assertWindowEthereum, MetaMask } from "@thirdweb-dev/wallets";
+
+export const metamaskWallet = () => {
+  let isInjected = false;
+  if (assertWindowEthereum(globalThis.window)) {
+    isInjected = !!globalThis.window.ethereum?.isMetaMask;
+  } else {
+    isInjected = false;
+  }
+
+  return {
+    id: MetaMask.id,
+    meta: MetaMask.meta,
```

Comment:
let's invert this, the object should drive the ID + meta and the wallet can inherit it / be passed it during construction time (so we do not need to import the full wallet class just to have the metadata object)